core/node/api_server/src/web3/namespaces/debug.rs

```diff
@@ -222,6 +318,6 @@ impl DebugNamespace {
             revert_reason,
             result.call_traces,
         );
-        Ok(Self::map_call(call, false))
+        Ok(Self::map_call(call, 0, hash, options))
```

Comment:
Just to clarify: Do we know what transaction hash / index geth returns for `debug_traceCall`? Is it zeroed, or skipped from the response?

Reply:
Its unpredictable and doesn't make much sense